src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -338,32 +344,42 @@
                                   :tx/bridge               constants/send-type-bridge
                                   constants/send-type-transfer)
          balances-per-chain (when token (:balances-per-chain token))
-         token-available-networks-for-suggested-routes
+         sender-token-available-networks-for-suggested-routes
          (when token
            (send-utils/token-available-networks-for-suggested-routes {:balances-per-chain
                                                                       balances-per-chain
                                                                       :disabled-chain-ids
-                                                                      disabled-from-chain-ids}))
+                                                                      disabled-from-chain-ids
+                                                                      :only-with-balance? true}))
+         receiver-token-available-networks-for-suggested-routes
+         (when token
+           (send-utils/token-available-networks-for-suggested-routes {:balances-per-chain
+                                                                      balances-per-chain
+                                                                      :disabled-chain-ids
+                                                                      disabled-from-chain-ids
+                                                                      :only-with-balance? false}))
          token-networks-ids (when token (mapv #(:chain-id %) (:networks token)))
-         sender-network-values (when token-available-networks-for-suggested-routes
+         sender-network-values (when sender-token-available-networks-for-suggested-routes
                                  (send-utils/loading-network-amounts
-                                  {:valid-networks     (if (= transaction-type :tx/bridge)
-                                                         (filter
-                                                          #(not= bridge-to-chain-id %)
-                                                          token-available-networks-for-suggested-routes)
-                                                         token-available-networks-for-suggested-routes)
+                                  {:valid-networks
+                                   (if (= transaction-type :tx/bridge)
+                                     (filter
+                                      #(not= bridge-to-chain-id %)
+                                      sender-token-available-networks-for-suggested-routes)
+                                     sender-token-available-networks-for-suggested-routes)
```

Comment:
Small comment, consider using `remove` instead of `filter ` + `not` + `predicate`.